internal/args/parse.go

```diff
@@ -24,18 +24,30 @@ func (f filterArgument) Validate() error {
 // IgnoredConfig defines arguments which can be accepted for compatibility with rsync,
 // but are ignored by exodus-rsync.
 type IgnoredConfig struct {
-	Recursive      bool `short:"r"`
-	Times          bool `short:"t"`
-	Delete         bool
-	KeepDirlinks   bool   `short:"K"`
-	OmitDirTimes   bool   `short:"O"`
-	Compress       bool   `short:"z"`
-	ItemizeChanges bool   `short:"i"`
-	Rsh            string `short:"e"`
-	CopyLinks      bool   `short:"L"`
-	Stats          bool
-	Timeout        int
-	Archive        bool `short:"a"`
+	Archive         bool `short:"a"`
+	Recursive       bool `short:"r"`
+	CopyLinks       bool `short:"L"`
+	KeepDirlinks    bool `short:"K"`
+	HardLinks       bool `short:"H"`
+	Perms           bool `short:"p"`
+	Executability   bool `short:"E"`
+	Acls            bool `short:"A"`
+	Xattrs          bool `short:"X"`
+	Owner           bool `short:"o"`
+	Group           bool `short:"g"`
+	Devices         bool
+	Specials        bool
+	DevicesSpecials bool   `short:"D" hidden:"1"`
```

Comment:
As mentioned in the commit message body, I wasn't able to find a practical way to flag both Devices and Specials using "-D" so it's wholly ignored. Apparently multiple arguments cannot share a flag, the first one parsed wins and the other ignored.

Reply:
I don't think this is quite right - I mean I get that maybe kong library doesn't have a trivial way to do this, but I don't think that's a good reason to just give up on passing this through to rsync.

Can't we do this with a couple of lines of code just after we call kong.Parse: if out.DevicesSpecials is true, then we set out.Devices true and out.Specials true.  There shouldn't be any problem then passing --devices and --specials through to rsync.  We would not literally be passing through the `-D` argument, instead we'd be "expanding" it, but I think that's OK.

Reply:
OK; done. I wasn't sure if this one flag was worth the additional, albeit minute, code.